### DISCUSSION TITLE
What former Republican US president said, ‘‘They say hard work never hurt anybody, but I figure why take the chance”?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#214] [Galaxy Trivia] [answered]
**Hint:** 40th

**Comment by Faiqlatheef** [comment, marked as answer]
The former Republican US President who said, "They say hard work never hurt anybody, but I figure why take the chance?" was Ronald Reagan. Reagan was known for his wit and humor and often used quips like this to lighten the mood during his speeches and interviews. The quote is often cited as an example of Reagan's folksy charm and ability to connect with ordinary Americans.
